ui_framework/components/pager/pager.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { KuiPagerButtons } from './pager_buttons';
+
+export function KuiPager({
+    className,
+    startNumber,
+    endNumber,
+    totalItems,
+    hasPreviousPage,
+    hasNextPage,
```

Comment:
That might not actually always be true.  For instance, the add panel pager:
<img width="522" alt="screen shot 2017-05-16 at 2 32 34 pm" src="https://cloud.githubusercontent.com/assets/16563603/26122160/8b1805d6-3a44-11e7-9400-6218ec6355f5.png">

endNumber < totalItems but hasNext is false.  likewise I could potentially see someone wanting to loop back around to the end if they kept hitting previous.